test/integration/install_scripts_int_test.go

```diff
@@ -50,6 +50,10 @@ func (suite *InstallScriptsIntegrationTestSuite) TestInstall() {
 			ts := e2e.New(suite.T(), false)
 			defer ts.Close()
 
+			// The home dir is set already for the test session, but we need to
+			// set it for the test so helper functions can find it.
+			suite.T().Setenv(constants.HomeEnvVarName, ts.Dirs.HomeDir)
```

Comment:
Nice find, but we should probably do this as part of `e2e.New()`. Otherwise we'll likely keep running into this.